packages/date-time-utilities/src/dateFormatting/dateFormatting.const.ts

```diff
@@ -0,0 +1,41 @@
+import { IDateGridStrings, IDateFormatting } from './dateFormatting.types';
+import { formatDay } from './formatDay';
+import { formatYear } from './formatYear';
+import { formatMonthDayYear } from './formatMonthDayYear';
+import { formatMonthYear } from './formatMonthYear';
+
+export const DEFAULT_LOCALIZED_STRINGS: IDateGridStrings = {
```

Comment:
Please could you extend this default object and interface with the other values from DEFAULT_STRINGS (packages/date-time/src/components/DatePicker/DatePicker.base.tsx:35)?

Reply:
Done